cg/constants/pacbio.py

```diff
@@ -31,3 +31,10 @@ class LoadingAttributesIDs:
     P_0: str = "loading_xml_report.productivity_0_n"
     P_1: str = "loading_xml_report.productivity_1_n"
     P_2: str = "loading_xml_report.productivity_2_n"
+
+
+class PolymeraseDataAttributeIDs:
+    MEAN_READ_LENGTH: str = "raw_data_report.read_length"
+    READ_LENGTH_N50: str = "raw_data_report.read_n50"
+    MEAN_LONGEST_SUBREAD_LENGTH: str = "raw_data_report.insert_length"
+    LONGEST_SUBREAD_LENGTH_N50: str = "raw_data_report.insert_n50"
```

Comment:
I have no clue what `N50` means, is it meaningful to include that in the names? Also, maybe this class should be an enum!

Reply:
https://en.wikipedia.org/wiki/N50,_L50,_and_related_statistics It is important to make the distinction between normal read length and N50 read length.

Reply:
The discussion about StrEnums, Enums and normal classes is still open and no conclusion has been reached https://github.com/Clinical-Genomics/atlas/pull/2485 I will leave this as it is for now.